File: PHP/cars/UberVan.php
<?php
    require_once('Car.php');
    class UberVan extends Car{
        private $typeCarAccepted;
        private $seatsMaterial;

        public function __construct($license,$driver,$typeCarAccepted,$seatsMaterial)
        {
            parent::__construct($license,$driver);
            $this->typeCarAccepted = $typeCarAccepted;
            $this->seatsMaterial = $seatsMaterial;
        }
        public function setPassenger($passenger){
            if($passenger==6){
            $this->passenger = $passenger;
            }
            else{
            echo "Debe tener 6 pasajeros \n";
            }
        }
        public function setTypeCarAccepted($TypeCar){
            $this->typeCarAccepted = $TypeCar;
        }
        public function setSeatsMaterial($seats){
            $this->seatsMaterial = $seats;
        }

        public function getTypeCarAccepted(){
            return $this->typeCarAccepted;
        }
        public function getSeatsMaterial(){
            return $this->seatsMaterial;
        }
    }
?>

File: Python/cars/UberVan.py
from cars.Car import Car
class UberVan(Car):
    __typeCarAccepted = []
    __seatsMaterial = []
    __passenger = int
    def __init__(self,license,driver,typeCarAccepted,seatsMaterial):
        super().__init__(license,driver)
        self.__typeCarAccepted = typeCarAccepted
        self.__seatsMaterial = seatsMaterial
    

    def setTypeCarAccepted(self,typeCar):
        self.__typeCarAccepted=typeCar

    def getTypeCarAccepted(self):
        return self.__typeCarAccepted
    
    def setSeatsMaterial(self,seats):
        self.__seatsMaterial=seats
    
    def getSeatsMaterial(self):
        return self.__seatsMaterial

    def setPassenger(self, passenger):
        if passenger == 6:
            self.__passenger=passenger
        else:
            print("Debe tener 6 pasajeros")

    def getPassenger(self):
        return self.__passenger